compiler/luci/pass/src/ResolveFormerCustomOpPass.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/ResolveFormerCustomOpPass.h"
+
+#include <luci/IR/CircleNodes.h>
+#include <luci/IR/AttrFusedActFunc.h>
+#include <luci/Profile/CircleNodeOrigin.h>
+
+#include <flatbuffers/flexbuffers.h>
+namespace
+{
+
+bool resolve_with_BroadcastTo(luci::CircleCustom *node)
+{
+  // check if input are support data types
+  auto input = loco::must_cast<luci::CircleNode *>(node->inputs(0));
+  if (input->dtype() != loco::DataType::FLOAT32)
+    return false;
```

Comment:
I just referred [this page](https://www.tensorflow.org/api_docs/python/tf/broadcast_to) but there seemed to not have any supported data types. 

> A Tensor. Has the same type as input.

We can just check if the data type of output is same with the one of the input feature map.